benchmarks/utils/api_mock.js

```diff
@@ -0,0 +1,51 @@
+const ApiMock = require('../../test/util/mimic/bitfinex_api_mock')
```

Comment:
use strict?

Reply:
Does `standard` support `use strict` rule checking?

Reply:
it looks like `standard` is no doesn't have a such rule and will not be added